src/adams_utils.jl

```diff
@@ -0,0 +1,42 @@
+function ϕ_and_ϕstar!(cache, dy, next_point, last_idx)
```

Comment:
Should these not have 2 space indentations to be in the style of the rest of the package?

Reply:
they should. Good catch